KsApi/models/graphql/adapters/GraphAPI.CreatePaymentSourceInput+CreatePaymentSourceInput.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+extension GraphAPI.CreatePaymentSourceInput {
+  static func from(_ input: CreatePaymentSourceInput) -> GraphAPI.CreatePaymentSourceInput {
+    return GraphAPI.CreatePaymentSourceInput(
+      paymentType: .creditCard,
```

Comment:
Is creditCard really the only payment source we accept? I wonder if debit cards are interpreted as credit card in this case?

Reply:
Yea I know it was strange, but the `GraphAPI` only supports `.creditCard` according to `line 367` on `GraphAPI`.